conformance/Makefile.am

```diff
@@ -2,7 +2,11 @@
 
 conformance_protoc_inputs =                                    \
   conformance.proto                                            \
-  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto
+  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto 
+  
+conformance_proto2_protoc_inputs =                             \
```

Comment:
Ah, a per generator limit.  Likely makes sense to add a comment here then to explain it to save someone else from trying it and seeing it fail.